src/tracker-component.js

```diff
@@ -178,6 +181,12 @@ export const Tracker = (config? : Config = {}) => {
     
     getOrCreateValidCartId();
 
+    if (config && config.user && config.user.id) {
+        setUserId(config.user.id);
+    } else {
+        getOrCreateValidUserId();
```

Comment:
let's wrap in a try/catch since it uses JSON.parse under the hood

Reply:
Fails gracefully now.